alembic/versions/967036f90ba1_gemini_model_py.py

```diff
@@ -0,0 +1,34 @@
+"""gemini_model.py
+
+Revision ID: 967036f90ba1
+Revises: 094ea67a2baf
+Create Date: 2018-02-27 11:15:29.518284
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '967036f90ba1'
+down_revision = '094ea67a2baf'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('osio_registered_repo',
```

Comment:
One more change here. The table names should be plural. So this should be `osio_registered_repo`.

Reply:
`osio_registered_repos`